debian/control

```diff
@@ -15,7 +15,8 @@ Build-Depends: cmake (>= 2.8.11),
                ninja-build,
                pandoc,
                pkg-config,
-               python,
+               python3-dev,
+               cython3,
```

Comment:
Please also depend on dh-python and sort these entries.

Reply:
done